frontend/src/app.tsx

```diff
@@ -6,14 +6,17 @@ import Sign from 'components/sign/sign'
 import NotFound from 'components/not-found/not-found';
 import Clinics from 'components/clinics/clinics';
 import DoctorsSearch from 'components/doctors-search/doctors-search';
+import DoctorDetails from 'components/doctor-details/doctor-details';
 
 const App: React.FC = () => (
   <Switch>
     <Route path={AppRoute.ADMIN_PAGE} exact component={AdminPage}/>
     <Route path={[AppRoute.SIGN_IN, AppRoute.SIGN_UP]} component={Sign}/>
     <Route path={AppRoute.CLINICS} component={Clinics} />
     <Route path={AppRoute.DOCTORS} component={DoctorsSearch}/>
+    <Route path="/details/:id" component={DoctorDetails} />
```

Comment:
```suggestion
    <Route path="/details/:id" component={DoctorDetails} />
```
Use AppRoute enum here